sql/storm-sql-core/src/jvm/org/apache/storm/sql/javac/CompilingClassLoader.java

```diff
@@ -125,7 +125,8 @@ public CompilingClassLoader(
     }
 
     /**
-     * @return Whether compilation was successful.
+     * Compiles source code to byte code.
```

Comment:
Wasn't this check disabled?

Reply:
We removed the rule to add missing Javadoc on public methods and classes. The rules to validate already present Javadoc were present and remained untouched. I think this makes sense because if Javadoc is present it should be valid. It could also be removed but deciding that slows down the PR workflow in my opinion.

Reply:
Makes sense, thanks.